### PR TITLE
Feature data lineage (table) #249

### PR DESCRIPTION
**Pull request by Rubik-W**
Detail: https://github.com/apache/incubator-dolphinscheduler/issues/249

### CODE REVIEW

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/SqlUtils.java [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+     * @return table List
+     * @throws UnsupportedDataTypeException
+     */
+    public static List<String> resolveSqlUpdateTables(DbType dbType, String sql) throws SQLException {
+        return resolveSqlTables(dbType, sql).get(Token.UPDATE);
+    }
+
+    /**
+     * 解析sql语句delete表名
+     *
```
Comment (Jave-Chen) [inline dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/SqlUtils.java:83]: Would you translate these comments into English ?

Comment (Rubik-W) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/SqlUtils.java:83]: fixed

**Comment by qiaozhanwei** [comment]
please resolve conflicting files . Thx

**Comment by Rubik-W** [comment]
For the new function of 1.3.1 iteration, the current PR has some deficiencies. After combing, re-raise the PR

**Pull request closed without merging**